C/CryptoMiniSat/bundled/Picosat.patch

```diff
@@ -0,0 +1,17 @@
+--- a/src/picosat/picosat.c
++++ b/src/picosat/picosat.c
+@@ -8144,10 +8144,13 @@ picosat_stats (PS * ps)
+            ps->prefix, picosat_max_bytes_allocated (ps) / (double) (1 << 20));
+ }
+
++#if defined(WIN32) || defined(_WIN32) || defined(__WIN32) || defined(WIN32_) || defined(WIN32__) || defined(__WIN32__) /* mingw does not include sys/resource.h */
```

Comment:
Why not just `_WIN32` which is the [only documented macro](https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170#microsoft-specific-predefined-macros) to identify a Windows target?  Anything else is wrong (C standard mandates that compiler should ***not*** define macros without a leading underscore) or custom, no need to cater for those, we should expect any well behaved Windows compiler defines `_WIN32`.

Also, this patch should probably be upstreamed.

Reply:
OK, done.